exonum-java-binding/core/rust/src/runtime/java_service_runtime.rs

```diff
@@ -198,20 +238,21 @@ impl ServiceFactory for JavaServiceFactory {
     }
 
     fn make_service(&mut self, context: &Context) -> Box<Service> {
-        let runtime = {
-            use exonum::helpers::fabric::keys;
-            let config: Config = context
-                .get(keys::NODE_CONFIG)
-                .expect("Unable to read node configuration.")
-                .services_configs
-                .get(super::cmd::EJB_CONFIG_SECTION_NAME)
-                .expect("Unable to read EJB configuration.")
-                .clone()
-                .try_into()
-                .expect("Invalid EJB configuration format.");
-            JavaServiceRuntime::get_or_create(config)
-        };
+        use exonum::helpers::fabric::keys;
+        let config: Config = context
+            .get(keys::NODE_CONFIG)
+            .expect("Unable to read node configuration.")
+            .services_configs
+            .get(super::cmd::EJB_CONFIG_SECTION_NAME)
+            .expect("Unable to read EJB configuration.")
+            .clone()
+            .try_into()
+            .expect("Invalid EJB configuration format.");
+        let runtime = { JavaServiceRuntime::get_or_create(config.clone()) };
 
-        Box::new(runtime.service_proxy().clone())
+        let artifact_id = runtime.load_artifact("");
```

Comment:
I suggest to add `artifact_path` field to the `JavaServiceFactory` and use it here instead of empty string. Later we will be able to create any amount of factories with different artifacts.